watson-developer-cloud/discovery-components
# feat: tooltip overlap pdf/text view

#### What do these changes do/fix?

Added tooltip for overlapping enrichments.

PDF view

<img width="1427" alt="image" src="https://github.com/watson-developer-cloud/discovery-components/assets/45341518/5e8e243d-0233-46fa-87c7-9c59d2b617d0">


Text view

<img width="1180" alt="image" src="https://github.com/watson-developer-cloud/discovery-components/assets/45341518/52c37c60-7cf2-4858-a70f-f181e168e66a">





#### How do you test/verify these changes?

1. Open a document in Discovery Tooling.
2. Select 2 enrichment categories (until overlap is available)
3. Hover over overlapping enrichments and verify tooltip content

Also, In Components sample app, test contracts visualization. Should be same as before. Some changes in the other views, required fixes in this view:
<img width="1173" alt="image" src="https://github.com/watson-developer-cloud/discovery-components/assets/45341518/adb94a22-a88a-4d51-be1e-ba5971907b5b">



#### Have you documented your changes (if necessary)?
NA

#### Are there any breaking changes included in this pull request?
No
